### ISSUE TITLE
10,000 stars

### ISSUE THREAD

**Issue by mxstbr** [mxstbr/react-boilerplate#1146]
I just want to say thank you to everybody who has ever starred, used, contributed, debugged or really done anything with react-boilerplate. The 10,000th person has just taken time out of their day to star this project on GitHub.

It's incredible to know that something I started has received so much recognition – but this isn't due to me, this is due to **you**. Without this amazing community, this would not have been possible.

This isn't _my_ boilerplate anymore, and it hasn't been for a long time. This is _our_ boilerplate.

Thank you so much for being a part of this. I'm so overwhelmed I don't really know what else to say. I love you all ❤

**Comment by gihrig** [comment]
Yes, it's a great product, but it's the superior community that keeps it that way.

Kudos to @mxstbr for being great community builder 🎉

**Comment by Altiano** [comment]
Looks like framework than boilerplate.. whaddya think? 💯

**Comment by KarandikarMihir** [comment]
Back when I started learning React, I had set a goal to be able to use this boilerplate one day. I've learned so many things from this project that I can't thank enough. 

My first ever pull request in open source was against this boilerplate. @mxstbr and other contributors guided me very patiently from time to time. 

I'm sure many others would be resonating with my thoughts. Ain't no sophisticated boilerplate than this one. Here's to another 10,000 stars. Cheers! :beers: :star:
